source/MRMesh/MRBestFitPolynomial.cpp

```diff
@@ -124,6 +124,39 @@ struct Solver<T, 4>
 };
 
 
+template <typename T, size_t degree>
+std::vector<std::complex<T>> solveWithFixedDegree( auto coeffs )
+{
+    assert( std::size( coeffs ) == degree + 1 );
```

Comment:
can it be `static_assert`?

Reply:
Maybe we could try to find a template magic to make it static (and eliminate recursion in this function), but it is not obvious to me.

Reply:
> it is not obvious to me

Besides copy-paste, of course :)